tests/manual-test-cases/Group5-Functional-Tests/5-5-Heterogeneous-ESXi.robot

```diff
@@ -47,7 +47,7 @@ Heterogenous ESXi Setup
 
     # Try again, if the VC failed quickly we might have enough time to try again
     ${pid-vc}=  Run Keyword Unless  ${status}  Deploy Nimbus vCenter Server Async  ${vc}
-    ${output}=  Run Keyword Unless  ${status}  Wait For Process  ${pid-vc}  timeout=40 minutes  on_timeout=terminate
+    ${output}=  Run Keyword Unless  ${status}  Wait For Process  ${pid-vc}  timeout=30 minutes  on_timeout=terminate
```

Comment:
since we have the wrapper retry now, I would actually get rid of this completely.  Let the first failure fail the setup and bounce back to the larger 10x retry.